libethcore/Precompiled.cpp

```diff
@@ -99,48 +99,77 @@ ETH_REGISTER_PRECOMPILED(identity)(bytesConstRef _in)
 
 // Parse _count bytes of _in starting with _begin offset as big endian int.
 // If there's not enough bytes in _in, consider it infinitely right-padded with zeroes.
-bigint parseBigEndianRightPadded(bytesConstRef _in, size_t _begin, size_t _count)
+bigint parseBigEndianRightPadded(bytesConstRef _in, bigint _begin, bigint _count)
```

Comment:
I think we should better pass `bigint`s by const reference, copying could be expensive.